samples/sensor/max44009/boards/frdm_k64f.overlay

```diff
@@ -1,13 +0,0 @@
-/*
```

Comment:
I think this file should be adjusted and moved to the other sample

Reply:
I don't think these "breadboarding required" overlay are actually very useful - they really only work for whoever put them together initiially based on whatever wiring they'd done on their bench. Plus, this binding is generic enough that there is nothing to learn from looking at a "real" example of it being used.
Happy to hear feedback from others though and make changes if that's the consensus. Thanks!

Reply:
True.
I assumed this sensor was on the board, which is not the case.